diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -257,3 +266,19 @@ fun ASTNode.areChildrenBeforeGroup(children: List<ASTNode>, group: List<ASTNode>
     return children.map { getChildren(null).indexOf(it) }.max()!! < group.map { getChildren(null).indexOf(it) }.min()!!
 }
 
+fun List<ASTNode>.checkOrder(getSiblingBlocks: ASTNode.() -> Pair<ASTNode?, ASTNode>,
```

Comment:
I don't like this method :( 

did you want to return anything by this method? 
if you are fixing, not checking - change the name of this method